### DISCUSSION TITLE
Remote start time

### DISCUSSION THREAD

**Discussion by Nickyes** [AntonyCorbett/OnlyT#444] [Q&A]
Hello, 
i've read that is possible to CONTROL time remotely, but i'm only able to visualize timer remotely.

How can i give this permission to other device (if it is possible)?

Thanks

**Comment by Nickyes** [comment]
Maybe something went wrong, now i can see the green button to start time. Maybe i've checked the wrong options.
But if you still would like to explain which could be the problem is better to understand to not repeat in future! :P

This give me an idea/request... i write now in another topic in right folder.

Thank you very much for this very precious tool!!

**Comment by AntonyCorbett** [comment]
@Nickyes Please refer #445
